src/modules/auth/component/SignUp/SignUp.tsx

```diff
@@ -1,20 +1,25 @@
 import {
   Grid,
 } from "@mui/material";
-import { ChangeEvent, FormEvent, useState } from "react";
+import {
+  ChangeEvent, FormEvent, useEffect, useState,
+} from "react";
 import { v4 as uuidv4 } from "uuid";
 import { PrimaryButton } from "../../../../components/UI/PrimaryButton";
 import { TextField } from "../../../../components/UI/TextField";
 import { Heading } from "../../../../components/UI/Heading";
-import { profileData, profilesData } from "../../../../data/data";
+import { profileData, token } from "../../../../data/data";
 import { Profile } from "../../../../types/types";
+import { useAppDispatch, useAppSelector } from "../../../../store/hooks";
+import { login } from "../../../../store/features/tokenSlice";
+import { addUser } from "../../../../store/features/userSlice";
 
 export const SignUp = () => {
-  const [formData, setFormData] = useState<Omit<Profile, "id">>(
+  const [formData, setFormData] = useState<Omit<Profile, "id" | "isLoggedIn">>(
     profileData,
   );
-
-  const [submittedData, setSubmittedData] = useState<Profile[]>([]);
+  const { profiles } = useAppSelector((state) => state.user);
```

Comment:
Again, do not use profiles in the redux. Use users in usermock class

Reply:
done